src/uImanpulate.js

```diff
@@ -0,0 +1,79 @@
+import getDataLS, { addTaskLS } from './localstorage.js';
+// import editText from './add-remove.js';
+
+export default function addItemUI(task) {
+  const list = document.querySelector('.list');
+  // Inside List create a div for list item
+  const listItem = document.createElement('div');
+  listItem.classList.add('list-item');
+  list.appendChild(listItem);
+  // Inside list item create a div label
+  const divlabel = document.createElement('div');
+  divlabel.classList.add('list-label');
+  divlabel.setAttribute('id', `box${task.index}`);
+  listItem.appendChild(divlabel);
+  // Inside list item create a div manipulate
+  const divman = document.createElement('div');
+  divman.classList.add('div-man');
+  listItem.appendChild(divman);
+  // Inside div label create a Input checkBox
+  const checkBox = document.createElement('input');
+  checkBox.classList.add('check');
+  checkBox.setAttribute('type', 'checkbox');
+  checkBox.setAttribute('name', `item${task.index}`);
+  checkBox.setAttribute('id', `item${task.index}`);
+  checkBox.checked = task.completed;
+  divlabel.appendChild(checkBox);
+  // Inside div label create a label
+  const labelItem = document.createElement('p');
+  labelItem.classList.add(`para-${task.index}`);
+  // labelItem.setAttribute('for', `item${task.index}`);
+  labelItem.textContent = `${task.description}`;
+  divlabel.appendChild(labelItem);
+  // Inside div manipulate create Delet button
+  const editBtn = document.createElement('span');
+  editBtn.classList.add('material-icons');
+  editBtn.classList.add('edit');
+  editBtn.setAttribute('id', `edit${task.index}`);
+  editBtn.textContent = 'edit';
+  divman.appendChild(editBtn);
+  // Inside div manipulate create Edit button
+  const deletBtn = document.createElement('span');
+  deletBtn.classList.add('material-icons');
+  deletBtn.classList.add('delete');
+  deletBtn.setAttribute('id', `delete${task.index}`);
+  deletBtn.textContent = 'delete';
```

Comment:
- [OPTIONAL] I would suggest adding a `cursor: pointer` CSS property to your `edit` and `delete` button to improve the UX of your app.

Reply:
Thank you